### DISCUSSION TITLE
Can os.js create .exe file ?

### DISCUSSION THREAD

**Discussion by WebAiz** [os-js/OS.js#810] [Q&A] [answered]
Hello, Im searching for desktop framework which able to provide me with node js context and generate .exe after build.
Can os.js provide me these functionalitites? Thank You

**Comment by ajmeese7** [comment]
What do you want an executable generated for?

**Reply by WebAiz** [reply]
I want a desktop app which will run on windows.

**Reply by ajmeese7** [reply]
A desktop app *of what*? Are you asking if OS.js can be compiled into an executable?

**Reply by zeroninefourfour** [reply]
> I want a desktop app which will run on windows.

i think you can use Electron

**Reply by WebAiz** [reply]
Thanks

**Comment by andersevenrud** [comment, marked as answer]
If you want to bundle your OS.js distro into an executable check out:  https://github.com/vercel/pkg

Disclaimer: I've never tried this before, but it's probably what you need.
